lib/tax_cloud/client.rb

```diff
@@ -40,7 +40,16 @@ def auth_params
       }
     end
 
-    def safe(&_block)
+    def client_params
+      { wsdl: TaxCloud::WSDL_URL }.tap do |params|
+        if TaxCloud.configuration
```

Comment:
I understand, but good opportunity to check. If all tests pass without this check, lets just go for it.